### PR TITLE
Feat/clients

### PR DESCRIPTION
**Pull request by katerinamartysko**

### CODE REVIEW

**Comment by katerinamartysko** [comment]
[Your task](https://trello.com/c/VokhfJgY/1-your-task)

**Comment by katerinamartysko** [comment]
[Create a JSON server & load data via axios](https://trello.com/c/ux5ttpnJ/2-create-a-json-server-load-data-via-axios)

File: src/index.tsx [hunk 1]
```diff
@@ -16,4 +14,4 @@ root.render(
 // If you want to start measuring performance in your app, pass a function
```
Comment (LexaLukovka) [inline src/index.tsx]: Remove unnecessary comment

File: package.json [hunk 2]
```diff
@@ -43,10 +45,12 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^5.42.0",
     "@typescript-eslint/parser": "^5.42.0",
-    "eslint": "8.22.0",
+    "eslint": "^8.22.0",
     "eslint-config-prettier": "^8.5.0",
     "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-react": "^7.31.10",
-    "prettier": "^2.7.1"
+    "husky": "^8.0.1",
```
Comment (LexaLukovka) [inline package.json]: If you don't use `husky` and `lint-staged`, remove them from dependencies. And also don't forget to remove from `package-lock.json`
```suggestion
    "husky": "^8.0.1",
    "lint-staged": "^13.0.3",
```

**Review by LexaLukovka — CHANGES_REQUESTED** [review]

File: src/index.tsx [hunk 3]
```diff
@@ -13,7 +11,4 @@ root.render(
   </React.StrictMode>
 );
 
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+
```
Comment (LexaLukovka) [inline src/index.tsx]: Remove empty line

File: src/index.tsx (hunk 1 above)
Comment (katerinamartysko) [reply src/index.tsx]: Done

File: package.json (hunk 2 above)
Comment (katerinamartysko) [reply package.json]: Done

**Review by LexaLukovka — APPROVED** [review]

**Pull request merged**